### PR TITLE
Fix swagger authentication for dotnet 5

### PR DESCRIPTION
**Pull request by Lucarine**
Hey @ttu,

I've put in the changes from #75 in this PR with some of the suggestions you've made on there.
I've left the xml comments in for swagger as some quick testing showed that those lines are still necessary.

Do let me know if there are some discrepancies between the PRs that I need to fix up 👍

### CODE REVIEW

File: FakeServer/Authentication/AuthenticationConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (ttu) [inline FakeServer/Authentication/AuthenticationConfiguration.cs:1]: Remove emptyline

File: FakeServer/Startup.cs [hunk 2]
```diff
@@ -133,11 +122,24 @@ public void ConfigureServices(IServiceCollection services)
 
                 if (useAuthentication)
                 {
-                    c.OperationFilter<AddAuthorizationHeaderParameterOperationFilter>();
+                    //c.OperationFilter<AddAuthorizationHeaderParameterOperationFilter>();
```
Comment (ttu) [inline FakeServer/Startup.cs:125]: Remove commented line

File: FakeServer/Startup.cs [hunk 3]
```diff
@@ -133,11 +122,24 @@ public void ConfigureServices(IServiceCollection services)
...
 
                     if (Configuration["Authentication:AuthenticationType"] == "token")
-                        c.DocumentFilter<AuthTokenOperation>();
+                    {
+                        //need token path for jwt definition
```
Comment (ttu) [inline FakeServer/Startup.cs:129]: This  comment can be also removed

File: FakeServer/Startup.cs [hunk 4]
```diff
@@ -133,11 +122,24 @@ public void ConfigureServices(IServiceCollection services)
...
+                        var tokenPath = TokenConfiguration.GetOptions().Value.Path;
+                        c.DocumentFilter<TokenOperation>();
+                        c.AddSecurityDefinition(JwtBearerDefaults.AuthenticationScheme, c.GetTokenSecurityDefinition(tokenPath));
+                        c.AddSecurityRequirement(c.GetTokenSecurityRequirement());
+                    }
+                    else
+                    {
+                        //Assumption: based on basic auth only
```
Comment (ttu) [inline FakeServer/Startup.cs:137]: This  comment can be also removed

**Review by ttu — APPROVED** [review]
Thank, nice work!

Just few small cleanups and then this is good to be merged. Thanks again!

File: FakeServer/Authentication/AuthenticationConfiguration.cs (hunk 1 above)
Comment (Lucarine) [reply FakeServer/Authentication/AuthenticationConfiguration.cs]: Removed

File: FakeServer/Startup.cs (hunk 2 above)
Comment (Lucarine) [reply FakeServer/Startup.cs]: Removed

File: FakeServer/Startup.cs (hunk 3 above)
Comment (Lucarine) [reply FakeServer/Startup.cs]: Removed

File: FakeServer/Startup.cs (hunk 4 above)
Comment (Lucarine) [reply FakeServer/Startup.cs]: Removed

**Comment by Lucarine** [comment]
Happy to oblige 😄

**Comment by ttu** [comment]
Thanks! If you have have any improvement ideas for the project, just let me know!

**Pull request merged**